base/bucket_gocb.go

```diff
@@ -1014,12 +1019,12 @@ func (bucket CouchbaseBucketGoCB) DeleteWithXattr(k string, xattrKey string) err
 		return err
 	}
 
-	_, err = bucket.Bucket.MutateInEx(k, gocb.SubdocDocFlagAccessDeleted, removeCas, 0).
+	_, deleteXattrErr := bucket.Bucket.MutateInEx(k, gocb.SubdocDocFlagAccessDeleted, removeCas, 0).
 		RemoveEx(xattrKey, gocb.SubdocFlagXattr).
 		Execute()
 
-	if err != nil && err != gocbcore.ErrSubDocSuccessDeleted {
-		return err
+	if deleteXattrErr != nil && deleteXattrErr != gocbcore.ErrSubDocSuccessDeleted {
```

Comment:
This is already documented with the error definition in gocbcore:
https://github.com/couchbase/gocbcore/blob/v6/error.go#L456

I feel like this is the right place for it.